### PR TITLE
use XFormBuilder instead of including a whole xform file

### PR DESCRIPTION
**Pull request by czue**
@kaapstorm this is my new favorite thing in the world

### CODE REVIEW

File: corehq/apps/userreports/tests/utils.py [hunk 1]
```diff
@@ -130,3 +131,17 @@ def mock_sql_backend():
...
+    xform = XFormBuilder()
+    xform.new_question('first_name', 'First Name', data_type='string')
+    xform.new_question('last_name', 'Last Name', data_type='string')
+    xform.new_question('children', 'Children', data_type='int')
+    xform.new_question('dob', 'Date of Birth', data_type='date')
+    xform.new_question('state', 'State', data_type='select', choices={
+        'MA': 'MA',
+        'MN': 'MN',
+        'VT': 'VT',
+    })
```
Comment (millerdev) [inline corehq/apps/userreports/tests/utils.py]: 79-line XForm represented in 10 lines of code. Nice compression ratio!

Comment (sravfeyn) [reply corehq/apps/userreports/tests/utils.py]: Awesome!

**Comment by czue** [comment]
anyone feel like approving this so i can merge it?

**Review by kaapstorm — APPROVED** [review]

**Comment by kaapstorm** [comment]
lol. Sorry, there you go.

**Pull request merged**